test/dashdb.columnName.test.js

```diff
@@ -56,15 +57,21 @@ describe('replaceById with custom column name for id', () => {
     const newBook = await Book.create({title: BOOK_TITLE});
     assert(newBook);
     assert.equal(newBook.title, BOOK_TITLE);
-    id = newBook.id;
-    const updatedBook = await Book.replaceById(id, {title: REVISED_BOOK_TITLE});
+    bookId = newBook.bookId;
```

Comment:
👍 I think this line is the key. Can we add a comment that: this shows that the property name is being correctly matched

Same for the test below